incubating/wrappers/java/src/test/java/io/seldon/wrapper/api/TestPredictionService.java

```diff
@@ -3,14 +3,23 @@
 import io.seldon.protos.PredictionProtos.DefaultData;
 import io.seldon.protos.PredictionProtos.SeldonMessage;
 import io.seldon.protos.PredictionProtos.Tensor;
+import io.seldon.wrapper.pb.ProtoBufUtils;
 import org.springframework.stereotype.Component;
 
 @Component
 public class TestPredictionService implements SeldonPredictionService {
   @Override
   public SeldonMessage predict(SeldonMessage payload) {
-    return SeldonMessage.newBuilder()
-        .setData(DefaultData.newBuilder().setTensor(Tensor.newBuilder().addShape(1).addValues(1.0)))
-        .build();
+    // echo payload back
+    return payload.toBuilder().build();
```

Comment:
more practical to return a copy of the payload back for testing. This makes sure that the payload is not deformed along the way and allows you to target particular pathways with different payloads.